install/homebrew.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: docs
+title: Homebrew installation
+permalink: /docs/install/homebrew/
+---
+This page describes how to run install Airsonic with Homebrew on macOS [Airsonic Homeprew tap](https://github.com/airsonic/homebrew-airsonic/).
+
+This installation method is designed for easy install and upgrades, and uses sane defaults. It might not work as well for very particular / special configurations.
+
+##### Prerequisites
+
+In order to install and Airsonic on macOS all you need is [Homebrew](https://brew.sh).
+
+Airsonic requires a recent Java runtime. The formula requires this cask, but it will not be installed automatically.
+
+```
+brew cask install java
+```
+
+All other dependencies will be installed and linked automatically.
+
+To easily load Airsonic, and start it with your system services, you should also install the [Homebrew Services](https://github.com/Homebrew/homebrew-services) tap.
+
+```
+brew tap homebrew/services
+```
+
+##### Installing
+
+The Airsonic formula currently resides in a custom project tap. It will be submitted to the official Homebrew formula repo at a future date.
+
+```
+brew tap airsonic/airsonic
+brew install airsonic
+```
+
+##### Upgrading
+
+```
+brew upgrade airsonic
+```
+
+##### Running Airsonic with Homebrew Services (preferred)
+
+```
+brew services start airsonic
+```
+
+##### Running Airsonic with launchd
+
+```
+cp `(brew --prefix)`/Cellar/airsonic/VERSION/homebrew.mxcl.airsonic.plist ~/Library/LaunchAgents/
+launchctl load -w ~/Library/LaunchAgents/homebrew.mxcl.airsonic.plist
+```
+
+> Replace VERSION with your Airsonic version
+
+##### Runtime settings used
+
+Here is a brief list of the runtime flags used, for reference.
+
+* `Xmx256m`
+* `Dlogging.file=#{var}/log/airsonic.log`
+* `Dlogging.level.root=ERROR`
+* `Dserver.host=0.0.0.0`
+* `Dserver.port=4040`
+* `Dserver.contextPath=/`
+* `Dairsonic.home=#{workingdir}`
+* `Dairsonic.host=0.0.0.0`
+* `Dairsonic.port=4040`
+* `Dairsonic.contextPath=/`
+* `Dairsonic.defaultMusicFolder=~/Music/iTunes/iTunes Media/Music`
+* `Dairsonic.defaultPodcastFolder=~/Music/iTunes/iTunes Media/Podcasts`
+* `Dairsonic.defaultPlaylistFolder=~/Music/Playlists`
```

Comment:
Those settings don't belong here !
Since we moved to spring boot, the airsonic.host/port are no longer used.
```
Dairsonic.host=0.0.0.0
Dairsonic.port=4040
```
Did you added them for an extra purpose ?

The airsonic.default*Folder are also no longer used since it leads to a database breakage when trying to change those values. ( muff1nman wanted to remove them , and until then we recommend not to use them ! )
```
Dairsonic.defaultMusicFolder=~/Music/iTunes/iTunes Media/Music
Dairsonic.defaultPodcastFolder=~/Music/iTunes/iTunes Media/Podcasts
Dairsonic.defaultPlaylistFolder=~/Music/Playlists
```
Again do you have any extra purpose for those ?

Looking at the hombrew repo those settings also are present in the .rb file, I advice also to clean this file from unused settings !

Reply:
@jooola thanks for reviewing.

```
Dairsonic.host=0.0.0.0
Dairsonic.port=4040
Dairsonic.contextPath=/
```

I just haven't updated for the new approach. Are you saying it's now _just_ `server.host` `server.port` and `server.contextPatch`? If so, I'll update both the formula the docs to suit.

Happy to drop the defaults. In general my thinking is just to make as many params as are sensible and useful explicit, rather than relying on some implicit values that people have trouble finding later. If there is no way to set the defaults right now I'll also remove from docs + formula.